Backend/dev/migrations-read-only/rider-app/route_stop_fare.sql

```diff
@@ -12,4 +12,102 @@ ALTER TABLE atlas_app.route_stop_fare ADD COLUMN vehicle_service_tier_id charact
 ALTER TABLE atlas_app.route_stop_fare ADD COLUMN vehicle_type text NOT NULL;
 ALTER TABLE atlas_app.route_stop_fare ADD COLUMN created_at timestamp with time zone NOT NULL default CURRENT_TIMESTAMP;
 ALTER TABLE atlas_app.route_stop_fare ADD COLUMN updated_at timestamp with time zone NOT NULL default CURRENT_TIMESTAMP;
-ALTER TABLE atlas_app.route_stop_fare ADD PRIMARY KEY ( end_stop_code, route_code, start_stop_code);
\ No newline at end of file
+ALTER TABLE atlas_app.route_stop_fare ADD PRIMARY KEY ( end_stop_code, route_code, start_stop_code);
+
+
+------- SQL updates -------
+
+
+--- Now DSL don't allow dropping tables instead we will drop not null constraint if any .Please be careful while running ---
+ALTER TABLE atlas_app.route_stop_fare ALTER COLUMN vehicle_variant DROP NOT NULL;
```

Comment:
vehicle_variant ADD column query is not there, why is Drop in this .sql file. Let's remove this and re-run.